{"name": "The user should be able to delete rate card sucessfully -- @1.1 ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=114.0.5735.199)\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x010BA813+48355]\n\t(No symbol) [0x0104C4B1]\n\t(No symbol) [0x00F55358]\n\t(No symbol) [0x00F3D293]\n\t(No symbol) [0x00F9E37B]\n\t(No symbol) [0x00FAC473]\n\t(No symbol) [0x00F9A536]\n\t(No symbol) [0x00F782DC]\n\t(No symbol) [0x00F793DD]\n\tGetHandleVerifier [0x0131AABD+2539405]\n\tGetHandleVerifier [0x0135A78F+2800735]\n\tGetHandleVerifier [0x0135456C+2775612]\n\tGetHandleVerifier [0x011451E0+616112]\n\t(No symbol) [0x01055F8C]\n\t(No symbol) [0x01052328]\n\t(No symbol) [0x0105240B]\n\t(No symbol) [0x01044FF7]\n\tBaseThreadInitThunk [0x76476839+25]\n\tRtlGetFullPathName_UEx [0x7708906F+1215]\n\tRtlGetFullPathName_UEx [0x7708903D+1165]\n\n", "trace": "  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\behave\\runner.py\", line 545, in run_hook\n    self.hooks[name](context, *args)\n  File \"Features\\environment.py\", line 75, in after_step\n    allure.attach(context.driver.get_screenshot_as_png(), name=\"screenshot\", attachment_type=AttachmentType.PNG)\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 965, in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 977, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 444, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 249, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "And Enter new \"Rate4\" in Rate Card name field", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed\nfrom unknown error: web view not found\n  (Session info: chrome=114.0.5735.199)\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x010BA813+48355]\n\t(No symbol) [0x0104C4B1]\n\t(No symbol) [0x00F55358]\n\t(No symbol) [0x00F3D293]\n\t(No symbol) [0x00F9E37B]\n\t(No symbol) [0x00FAC473]\n\t(No symbol) [0x00F9A536]\n\t(No symbol) [0x00F782DC]\n\t(No symbol) [0x00F793DD]\n\tGetHandleVerifier [0x0131AABD+2539405]\n\tGetHandleVerifier [0x0135A78F+2800735]\n\tGetHandleVerifier [0x0135456C+2775612]\n\tGetHandleVerifier [0x011451E0+616112]\n\t(No symbol) [0x01055F8C]\n\t(No symbol) [0x01052328]\n\t(No symbol) [0x0105240B]\n\t(No symbol) [0x01044FF7]\n\tBaseThreadInitThunk [0x76476839+25]\n\tRtlGetFullPathName_UEx [0x7708906F+1215]\n\tRtlGetFullPathName_UEx [0x7708903D+1165]\n\n", "trace": "  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\behave\\runner.py\", line 545, in run_hook\n    self.hooks[name](context, *args)\n  File \"Features\\environment.py\", line 75, in after_step\n    allure.attach(context.driver.get_screenshot_as_png(), name=\"screenshot\", attachment_type=AttachmentType.PNG)\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 965, in get_screenshot_as_png\n    return b64decode(self.get_screenshot_as_base64().encode(\"ascii\"))\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 977, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\"value\"]\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 444, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\akankshyad\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 249, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1717046725805, "stop": 1717046725851}, {"name": "And click on save button for rates", "status": "skipped", "start": 1717046725852, "stop": 1717046725852}, {"name": "And verify the \"Rate card updated successfully.\"", "status": "skipped", "start": 1717046725852, "stop": 1717046725852}, {"name": "And import the rate card file", "status": "skipped", "start": 1717046725852, "stop": 1717046725852}, {"name": "And delete the rate card", "status": "skipped", "start": 1717046725853, "stop": 1717046725853}, {"name": "And verify the \"Rate version(s) deleted successfully.\"", "status": "skipped", "start": 1717046725853, "stop": 1717046725853}], "parameters": [{"name": "Name", "value": "Rate4"}, {"name": "message", "value": "Rate card updated successfully."}, {"name": "message1", "value": "Rate version(s) deleted successfully."}], "start": 1717046724792, "stop": 1717046725853, "uuid": "7515ceb7-dbcb-48e4-90e2-2f42da8edb58", "historyId": "b4b76b9d87cfccdd81656d092c3df489", "fullName": "Verify the TMS functionality: The user should be able to delete rate card sucessfully", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "Rate_Cards_Delete"}, {"name": "feature", "value": "Verify the TMS functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}